diff --git a/therubyracer.gemspec b/therubyracer.gemspec
index 5a6ace7..95e2a27 100644
--- a/therubyracer.gemspec
+++ b/therubyracer.gemspec
@@ -8,9 +8,8 @@ Gem::Specification.new do |gem|
   gem.description   = "Call JavaScript code and manipulate JavaScript objects from Ruby. Call Ruby code and manipulate Ruby objects from JavaScript."
   gem.homepage      = "http://github.com/cowboyd/therubyracer"
 
-  gem.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
-  gem.files         = `git ls-files`.split("\n")
-  gem.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
+  gem.files         = `find . -type f |sort`.split("\n")
+  gem.test_files    = `find spec -type f |sort`.split("\n")
   gem.name          = "therubyracer"
   gem.extensions    = ["ext/v8/extconf.rb"]
   gem.require_paths = ["lib", "ext"]
