diff --git a/listen.gemspec b/listen.gemspec
index 4e110b0..cda047b 100644
--- a/listen.gemspec
+++ b/listen.gemspec
@@ -14,9 +14,8 @@ Gem::Specification.new do |s|
   s.description = 'The Listen gem listens to file modifications and '\
     'notifies you about the changes. Works everywhere!'
 
-  s.files = `git ls-files -z`.split("\x0").select do |f|
-    %r{^(?:bin|lib)\/} =~ f
-  end + %w(CHANGELOG.md CONTRIBUTING.md LICENSE.txt README.md)
+  s.files = `find bin lib -type f |sort`.split("\n") +
+            %w(CHANGELOG.md CONTRIBUTING.md LICENSE.txt README.md)
 
   s.test_files   = []
   s.executable   = 'listen'
